neural_transport/models/flowmatching.py

```diff
@@ -11,7 +11,7 @@
 from flow_matching.solver import ODESolver
 
 # neural_transport
-from neural_transport.models import MODELS
+from neural_transport.models.unet import UNet
```

Comment:
i'd suggest to keep the MODELS, so later we can easily exchange UNet for another model